src/includes/capture-error/javascript.remix.mdx

```diff
@@ -0,0 +1,11 @@
+You can pass an `Error` object to `captureException()` to get it captured as event. It's also possible to pass non-`Error` objects and strings, but be aware that the resulting events in Sentry may be missing a stacktrace.
```

Comment:
I know you copied this text from an existing include, but we should still fix this stuff anyway. I'll note it as something I'll look to fix in existing includes.

```suggestion
You can pass an `Error` object to `captureException()` to have it captured as an event. It's also possible to pass non-`Error` objects and strings, but be aware that the resulting events in Sentry may be missing a stack trace.
```